### PR TITLE
Added basic CI/CD with Github actions

### PR DESCRIPTION
**Pull request by jdhouti**

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-JSON_PATH = './fyi_links.json'
-NETFLIX_SALARY_LINK = 'salary_link'
-RECENT_SALARY_XPATH = 'salary_xpath'
-RECENT_YOE_XPATH = 'yoe_xpath'
-RECENT_LEVEL_NAME_XPATH = 'level_xpath'
```
Comment (Mohit-Doshi) [inline constants.py:5]: LOL I always thought Python standard was single-quotes

**Comment by Mohit-Doshi** [comment]
LGTM 🚢 🚢 🚢

**Review by Mohit-Doshi — APPROVED** [review]

File: constants.py (hunk 1 above)
Comment (jdhouti) [reply constants.py:5]: I think it depends on the context

Comment (tainhenning) [reply constants.py:5]: it is single :^)

Comment (jdhouti) [reply constants.py:5]: According to PEP, no preference. Just pick one and stick to it I guess? 

https://www.python.org/dev/peps/pep-0008/#string-quotes

**Pull request merged**